gravitee-apim-portal-webui-next/src/components/nav-bar/desktop-nav-bar/desktop-nav-bar.component.html

```diff
@@ -26,9 +26,27 @@
         </a>
       }
       @default {
-        <a [routerLink]="['/documentation', navigationItem.id]" mat-stroked-button color="primary" class="menu-items__custom_link">
-          {{ navigationItem.title }}
-        </a>
+        <div routerLinkActive #rla="routerLinkActive">
+          @if (rla.isActive) {
+            <a
+              [routerLink]="['/documentation', navigationItem.id]"
+              routerLinkActive
+              mat-flat-button
+              color="primary"
+              class="menu-items__custom_link">
+              {{ navigationItem.title }}
+            </a>
```

Comment:
You can reuse the component above:
```suggestion
<app-nav-bar-button [path]="['/documentation', navigationItem.id]">{{ navigationItem.title }}</app-nav-bar-button>
```

Reply:
Done